backend/app/routers/matching.py

```
@@ -1,3 +1,36 @@
+    current_user: User = Depends(get_current_user),
+    db: AsyncSession = Depends(get_db),
+):
+    if current_user.role != "admin":
+        raise HTTPException(status_code=403, detail={"error": "UNAUTHORIZED"})
+    return await matching_service.run_matching(db)
+
+
+@router.patch("/relay-chains/{chain_id}/approve")
+async def approve_chain(
+    chain_id: int,
+    current_user: User = Depends(get_current_user),
```

Ruff (0.15.10)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)